Objective-C/Tests/LogTest.m

```diff
@@ -46,44 +36,23 @@ @implementation LogTest {
 
 - (void) setUp {
     [super setUp];
-    [self backupLoggerConfig];
     NSString* folderName = [NSString stringWithFormat: @"LogTestLogs_%d", arc4random()];
     logFileDirectory = [NSTemporaryDirectory() stringByAppendingPathComponent: folderName];
+    _backup = CBLLogSinks.file;
 }
 
 - (void) tearDown {
     [super tearDown];
     [[NSFileManager defaultManager] removeItemAtPath: logFileDirectory error: nil];
-    [self restoreLoggerConfig];
-}
-
-- (CBLLogFileConfiguration*) logFileConfig {
-    return [[CBLLogFileConfiguration alloc] initWithDirectory: logFileDirectory];
-}
-
-- (void) backupLoggerConfig {
-    _backup = [[FileLoggerBackup alloc] init];
-    _backup.level = CBLDatabase.log.file.level;
-    _backup.config = CBLDatabase.log.file.config;
-    _backupConsoleLevel = CBLDatabase.log.console.level;
-    _backupConsoleDomain = CBLDatabase.log.console.domains;
-}
-
-- (void) restoreLoggerConfig {
-    if (_backup) {
-        CBLDatabase.log.file.level = _backup.level;
-        CBLDatabase.log.file.config = _backup.config;
-        _backup = nil;
-    }
-    CBLDatabase.log.custom = nil;
-    CBLDatabase.log.console.level = _backupConsoleLevel;
-    CBLDatabase.log.console.domains = _backupConsoleDomain;
-    
+    CBLLogSinks.file = _backup;
+    CBLLogSinks.console = [[CBLConsoleLogSink alloc] initWithLevel: kCBLLogLevelWarning];
```

Comment:
Should set to a back up. For example, we want to debug an issue by turning a console log at DEBUG level on in the base class's setup, this setting will just turn it back to WARNING.